open-metadata-implementation/view-services/tex-view/tex-view-server/src/main/java/org/odpi/openmetadata/viewservices/tex/server/TexViewRESTServices.java

```diff
@@ -42,6 +47,49 @@ public TexViewRESTServices() {
     }
 
 
+    /**
+     * Retrieve platform origin
```

Comment:
what is the platform origin.

Reply:
It is an identifier that is returned by the server-platform-origin REST API in the platform-services API.
Typically it looks a lot like this...
ODPi Egeria OMAG Server Platform (version 2.4-SNAPSHOT)